MySQLdb/connections.py

```diff
@@ -162,6 +169,14 @@ class object, used to create cursors (keyword only)
 
         kwargs2['client_flag'] = client_flag
 
+        if 'ssl_mode' in kwargs:
+            if not _mysql.get_have_enum_mysql_opt_ssl_mode():
+                raise NotSupportedError('Does not support ssl_mode specification: %s' % _mysql.get_client_info())
+            if hasattr(SSL_MODE, kwargs['ssl_mode']):
+                kwargs2['ssl_mode'] = getattr(SSL_MODE, kwargs['ssl_mode'])
+            else:
+                raise NotSupportedError('Unknown MySQL ssl_mode specification: %s' % kwargs['ssl_mode'])
+
```

Comment:
Implement this block in C.

Reply:
Implemented error checking on the C lang side.
And removed MySQLdb.constants.SSL_MODE, which is no longer needed.